modular_chomp/maps/southern_cross/overmap/space/fueldepot.dmm

```diff
@@ -1176,12 +1180,12 @@
 	dir = 1
 	},
 /obj/structure/window/reinforced{
-	health = 1e+006;
+	health = 1e+06;
 	req_access = list(5)
```

Comment:
Another set of weird windows which are here for no reason but fixing these is out of scope for today.

Why the fuck this window has req_access = list(5) on it, I have no idea.